odoo/addons/base/models/ir_qweb_fields.py

```diff
@@ -517,6 +518,31 @@ def value_to_html(self, value, options):
         return format_duration(value)
 
 
+class TimeConverter(models.AbstractModel):
+    """ ``time`` converter, to display integral or fractional value as
+    human-readable time (e.g. 1.5 as "1:30 AM").
+
+    Can be used on any numerical field between [0:24[
+    """
+    _name = 'ir.qweb.field.time'
+    _description = 'QWeb Field Time'
+    _inherit = 'ir.qweb.field'
+
+    @api.model
+    def value_to_html(self, value, options):
+        if value < 0:
+            raise ValueError(_("The value passed should be positive"))
+        time_vals = format_duration(value).split(':')
+        if int(time_vals[0]) > 23:
+            raise ValueError(_("The hour must be between 0 and 23"))
+        t = time(hour=int(time_vals[0]), minute=int(time_vals[1]))
+
+        locale = babel_locale_parse(self.user_lang().code)
+        pattern = options.get('format', 'short')
+
+        return babel.dates.format_time(t, format=pattern, tzinfo=None, locale=locale)
```

Comment:
Isn't it possible to have some unit tests ? Just trying to see the output of value_to_html.

Reply:
Test added for the output of the new field